RELEASE_NOTES.rst

```diff
@@ -40,14 +40,16 @@ Highlights:
   new = lmdb.open("dataset_db.mdb", subdir=False, map_size=2**30)
   with new.begin(write=True) as txn:
     for key, value in old.items():
-      txn.put(key.encode(), pyon.encode(value).encode())
+      txn.put(key.encode(), pyon.encode((value, {})).encode())
   new.close()
 
 Breaking changes:
 
 * ``SimpleApplet`` now calls widget constructors with an additional ``ctl`` parameter for control
   operations, which includes dataset operations. It can be ignored if not needed. For an example usage,
   refer to the ``big_number.py`` applet.
+* Metadata such as units and precision can now be associated with a dataset. 
+  The ``(value, {})`` is needed to enable metadata in persistent datasets.
```

Comment:
removing :+1:

Reply:
For a breaking change like this, it would be very useful to document what the necessary changes in user code are (in this case, to clients that consume the dataset db using the `sync_struct` interface).